src/scripts/main.js

```diff
@@ -1 +1,69 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.querySelector('body').addEventListener('click', function() {
+    resolve();
+  });
+
+  setTimeout(() => reject(new Error('promise was rejected')), 3000);
+});
+
+firstPromise.then(() => {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.className = 'success';
+  div.innerText = 'First promise was resolved';
+  document.body.append(div);
+});
+
+firstPromise.catch(() => {
+  const div = document.createElement('div');
+
+  div.setAttribute('data-qa', 'notification');
+  div.className = 'warning';
+  div.innerText = 'First promise was rejected';
+  document.body.append(div);
+});
```

Comment:
```suggestion
firstPromise
  .then(() => {
  const div = document.createElement('div');

  div.setAttribute('data-qa', 'notification');
  div.className = 'success';
  div.innerText = 'First promise was resolved';
  document.body.append(div);
})
  .catch(() => {
  const div = document.createElement('div');

  div.setAttribute('data-qa', 'notification');
  div.className = 'warning';
  div.innerText = 'First promise was rejected';
  document.body.append(div);
});
```

Reply:
you can do it in a chain

also create a separate function for a notification and pass needed data there